app/MMB/src/scenes/tripServices/transportation/Address.js

```diff
@@ -0,0 +1,75 @@
+// @flow
+
+import * as React from 'react';
+import { View, Platform } from 'react-native';
+import { StyleSheet, Text, Touchable } from '@kiwicom/mobile-shared';
+import { Translation } from '@kiwicom/mobile-localization';
+import { defaultTokens } from '@kiwicom/mobile-orbit';
+
+type Coordinate = {|
+  +latitude: number,
+  +longitude: number,
+|};
+
+type AddressType = {|
+  +location: Coordinate,
+  +name: string | null,
+  +address: string,
+|};
+
+type Props = {|
+  +address: AddressType,
+  +setLocation: (location: Coordinate) => void,
+|};
+
+export default class Address extends React.Component<Props> {
+  setLocation = () => {
+    this.props.setLocation(this.props.address.location);
+  };
+  render() {
+    return (
+      <Touchable onPress={this.setLocation}>
+        <View style={styles.wrapper}>
+          {Platform.OS === 'ios' &&
```

Comment:
Will there be some code for android here also?

Reply:
Last time I could check on an Android phone, there was no title being displayed, only the address.